packages/web/src/scss/components/UNSTABLE_EmptyState/index.html

```diff
@@ -3,87 +3,63 @@
 <section class="docs-Section">
   <h2 class="docs-Heading">Default</h2>
   <div class="docs-Stack docs-Stack--stretch">
+    <div style="display: flex; flex-direction: column; align-self: center; gap: 64px; max-width: 582px;">
```

Comment:
If you decide to change the structure as outlined [here](https://github.com/lmc-eu/spirit-design-system/pull/1481/files#r1643916861), this `<div>` can go away.

```suggestion
```